src/components/App/App.tsx

```diff
@@ -41,14 +40,23 @@ class App extends React.Component<{}, IState> {
   
   render() {  
       return (
-        <div className="App">
-          <NavBar />
-          <RandomContainer 
-            randomDrinks={this.state.drinkArray}     
-          />
+      <div>
```

Comment:
Good to see and learn that most Route logic will live/start in the App component